README.CN.md

```diff
@@ -156,9 +156,13 @@ const cherryInstance = new Cherry({
 // 引入Cherry引擎核心构建
 // 引擎配置项与Cherry通用，以下文档内容仅介绍Cherry核心包
 import CherryEngine from 'cherry-markdown/dist/cherry-markdown.engine.core';
+import Engine from 'cherry-markdown/dist/types/Engine';
+
 const cherryEngineInstance = new CherryEngine({
   value: '# welcome to cherry editor!',
-});
+}) as Engine;
```

Comment:
A good solution to fix typings in the current version. BTW, we're still dealing with type checking issues. PRs for these issues are welcome!